### ISSUE TITLE
Verify test case 737 (and those like it)

### ISSUE THREAD

**Issue by joanmarie** [WhatSock/w3c-alternative-text-computation#7]
AccName test case 737 has the following:

```
if given
  <label for="test">
    crazy
    <select name="member" size="1" role="menu" tabindex="0">
      <option role="menuitem" value="beard" selected="true">clown</option>
      <option role="menuitem" value="scuba">rich</option>
    </select>
  </label> 
  <input type="image" id="test" src="foo.jpg"/>
then the accessible name of the element with id of "test" is "crazy clown"
```

That test case can be found here and confirms that expectation: https://whatsock.github.io/w3c-alternative-text-computation/Autogenerated%20AccName%201.1%20Testable%20Statements%20-%20W3C/Name%20test%20case%20737.html

But most user agents are only returning a name of "crazy." Looking at the AccName spec, I see that step 2.E's third bullet states:

> If the embedded control has role `combobox`, return the text alternative of the chosen option.

And if I change the test case to have role `combobox` the user agents do the right thing. Therefore, could you please recheck your expected result and confirm it is correct or correct it? Also, if you correct it, could you please update your test cases so that following the link I mentioned above reflects the new expectation? I'll then update the AccName tests accordingly.

Thanks in advance!

**Comment by joanmarie** [comment]
Those like 737 include: 548, 733, 734, 735, and 736

**Comment by accdc** [comment]
This was fixed in version 1.19, and includes logic to recognize when combobox is present instead of other widget roles.

**Comment by joanmarie** [comment]
Hurray and thanks!! I've updated the wiki. And once I go through the status of other open issues, I'll generate new tests and update WPT.

Closing this issue as fixed.
